usecases/sharding/state.go

```diff
@@ -127,22 +127,23 @@ type nodes interface {
 }
 
 func InitState(id string, config Config, nodes nodes, replFactor int64, partitioningEnabled bool) (*State, error) {
+	names := nodes.Candidates()
+	if f, n := replFactor, len(names); f > int64(n) {
+		return nil, fmt.Errorf("not enough replicas: found %d want %d", n, f)
```

Comment:
Could it make sense to change it to

```
Not enough storage candidates for replication: found %d want %d
```

just to make the error message clearer for users

Reply:
Agreed that a clearer message is possible :) done